kafka-connect-neo4j/docker/readme.adoc

```diff
@@ -88,4 +88,39 @@ and click to the **Sink** tab. You must find a table just like this:
 |my-topic
 |===
 
-Now you can send your AVRO data to the `my-topic` topic
+# Use the data generator
```

Comment:
can you fix this to use asciidoc syntax?
i.e. == for headers, ---- for listings

remove the $ for  command line invocations, so it's easier to copy and paste

Reply:
done!